src/tools/ImageConverter.cpp

```diff
@@ -0,0 +1,21 @@
+// Created by Maxandre Ogeret.
+// (c) 2022 University of Tartu - Autonomous Driving Lab.
+
+#include "tools/ImageConverter.h"
+
+ImageConverter::ImageConverter(DriveworksApiWrapper* driveworksApiWrapper, uint32_t width, uint32_t height,
```

Comment:
Please add some comments what it does and why it is needed.

Reply:
class has been removed.